cmake/generateFilelistEngine.sh

```diff
@@ -0,0 +1,17 @@
+echo  "set( engine_sources"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=4401329)